remoteworkitem/scheduler.go

```diff
@@ -11,11 +12,13 @@ import (
 
 // TrackerSchedule capture all configuration
 type trackerSchedule struct {
-	TrackerID   int
-	URL         string
-	TrackerType string
-	Query       string
-	Schedule    string
+	TrackerID      int
+	URL            string
+	TrackerType    string
+	TrackerQueryID int
+	Query          string
+	Schedule       string
+	LastUpdated    *time.Time
```

Comment:
Not a big thing but this is more of "LastRun" than "LastUpdated" because the latter might give a feeling that it is the update of the schedule we are tracking here in this field.